integration-tests/test_connect.py

```diff
@@ -18,6 +18,7 @@
 )
 
 logger = logging.getLogger(__name__)
+pytestmark = pytest.mark.usefixtures("install_katello_rpm")
```

Comment:
I would suggest that instead of using it in all test scripts, try autouse=True in fixture  "install_katello_rpm"